Makefile

```diff
@@ -168,7 +168,7 @@ run-all-unit-test: test_smartstreams
 	cargo test -p fluvio-storage $(RELEASE_FLAG) $(TARGET_FLAG)
 	make test-all -C src/protocol	
 
-run-unstable-test:	build_filter_wasm
+run-unstable-test:	test_smartstreams
```

Comment:
don't run smart `smartstream` test for integration test.  Only test with ignore should be run.  This is why there is failure

Reply:
Oh, hm. It just looked like the `build_filter_wasm` was there previously which tested the smart streams so I thought I was just updating this. Doesn't one of the tests require the .wasm file from the old smart_filter crate?

Reply:
Yes, smart filter is dependent but you don't need to run tests in those crates

Reply:
Ok, I'll make an independent task for building